feat: Supplier Sourced Item in Subcontracting (#172)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/manufacturing/subcontracting.md

<img class="screenshot" alt="Sub-Contracting" src="{{docs_base_url}}/assets/img/manufacturing/subcontract5.png">

### 1.5 Supplier Sourced Raw Material
While creating a BOM for subcontracting, there might be raw material which Supplier will have to procure themselves like nuts and bolts.

> While creating a BOM for subcontracting, there might be raw material which Supplier will have to procure themselves like nuts and bolts. While creating a Stock Entry for Transfer from Purchase Order, these items can be excluded one by one but it is impossible to do so if you have more than 100 items. 

Reviewer: ```suggestion
While creating a Stock Entry for "Transfer" from Purchase Order, these items can be excluded one by one, but it is impossible to do so if you have more than 100 items. 
```

> If some raw material is sourced by the Supplier directly, such raw material has to be marked in the BOM.

Reviewer: by marked do you mean added in the BOM? If yes, then write as follows:
```suggestion
If some raw material is sourced by the Supplier directly, then such raw material has to be included in the BOM.
```

> * It will have zero value in BOM

Reviewer: what is value? Rate or qty? Mention that clearly

* In Purchase Order, this raw material will not appear in Supplied Items since it is not supplied

> * And also while creating a Transfer, such items will be excluded from Stock Entry

Reviewer: ```suggestion
* Also, while creating a "Transfer", such items will be excluded from the Stock Entry
```

<img class="screenshot" alt="Supplier Sourced Raw Material" src="{{docs_base_url}}/assets/img/manufacturing/supplier_sourced_subcontracting.png">

> However, the Supplier may choose to include the supplier provided items in thier Sales Order sent to you.

Reviewer: ```suggestion
However, the Supplier may choose to include the supplier-provided items in their Sales Order sent to you.
```
